packages/common/types/config.ts

```diff
@@ -44,13 +44,13 @@ export type Drm = {
 };
 
 export type PlaylistType = keyof typeof PLAYLIST_TYPE;
-
-export type PlaylistMenuType = Extract<PlaylistType, 'playlist' | 'content_list'>;
+export type PlaylistMenuType = Extract<PlaylistType, 'playlist' | 'content_list' | 'media'>;
```

Comment:
Well, it looks like it is not only about playlists now. I would suggest the following:

```
PLAYLIST_TYPE -> APP_CONTENT_TYPE
PlaylistType -> AppContentType
PlaylistMenuType -> AppMenuType
PlaylistShelfType -> AppShelfType
```